app/utils/callback/lava.py

```diff
@@ -480,6 +480,9 @@ def _add_test_results(group, results):
         reference = test_meta.get("reference")
         if reference:
             test_case[models.ATTACHMENTS_KEY] = [reference]
+        test_case_log_line = test.get("log_start_line")
```

Comment:
Please rebase on top of #214 as it works out the line numbers.  In spite of its name, `log_start_line` is _not_ actually the line number where the test case started.